test/elixir/test/all_docs_test.exs

```diff
@@ -242,13 +262,17 @@ defmodule AllDocsTest do
     resp = Couch.post(
       "/#{db_name}/_all_docs",
       body: %{
-        :keys => [1, 2],
-        :limit => 1
+        :keys => ["1", "2"],
```

Comment:
Why did we have to change the keys to strings here? Is there an API change here that we should make a note of?

Reply:
I notice the previous test is still using integer keys so I assume those still work. The addition of testing include_docs is nice, but I'd prefer to not change unrelated bits if possible.

Reply:
The original test was just returning `%{"error" => "not_found", "key" => 1}` which is not quite what we should be testing. We should be testing we return the document for that key. So I've added in the test to check that we return an error for the incorrect key and a proper test to prove we return keys. No functionality changed. Just improving the tests.

Reply:
Ah! Sounds good. That certainly wasn't testing what I thought it was testing by reading it.